cli/azd/pkg/ext/hooks_manager_test.go

```diff
@@ -103,7 +103,7 @@ func ensureScriptsExist(t *testing.T, configs map[string]*HookConfig) {
 	for _, hook := range configs {
 		ext := filepath.Ext(hook.Run)
 
-		if ext != "" {
+		if ext != "" && len(ext) <= 5 {
```

Comment:
can you add a constant to describe the meaning of `5` ?

Reply:
Updated to make it more clear what is happening.  We are just validating the file extension.